configure.ac

```diff
@@ -1048,6 +1048,64 @@ if test "X$HDF5_TOOLS" = "Xno"; then
   echo "Building HDF5 tools is disabled"
 fi
 
+## ----------------------------------------------------------------------
+## Check if they would like to enable building doxygen files
+##
+
+## This needs to be exposed for the library info file.
+AC_SUBST([HDF5_DOXYGEN])
+
+## Default is to build DOXYGEN
+HDF5_DOXYGEN=yes
+
+AC_MSG_CHECKING([if building doxygen is disabled])
```

Comment:
Is it checking if building doxygen is disabled or enabled?   Message doesn't match line 1052.  What does "no" in line 1105 mean?  Not enabled, or not disabled?

Should building doxygen be on by default or off?

Reply:
Fixed